ocaml/xapi/xapi_pool_update.ml

```diff
@@ -259,6 +260,13 @@ let parse_update_info xml =
       with
       | _ -> []
     in
+    let enforce_homogeneity =
+      try
+        List.assoc "enforce-homogeneity" attr
+        |> fun s -> s = "1" || String.lowercase s = "true"
```

Comment:
I know. I am just saying that having only one copy of this function to call will make it easier to fix the deprecation in the future (and also I like to reduce duplication)